tmail-backend/apps/distributed/src/main/conf/jvm.properties

```diff
@@ -5,7 +5,7 @@
 # my.property=whatever
 
 # Required to locate Cassandra driver configuration
-config.file=conf/cassandra-driver.conf
+config.file=/root/conf/cassandra-driver.conf
```

Comment:
I have checked and on tmail.linagora.com, TWP env and CNB we use `config.file=/root/conf/cassandra-driver.conf` too.

The thing I do not understand: on MU we set path for `jvm.properties`:
```yaml
    environment:
      - JAVA_TOOL_OPTIONS= ... -Dextra.props=/root/conf/jvm.properties
```

Then we use relative path to load `cassandra-driver.conf` in `jvm.properties`:
```
config.file=conf/cassandra-driver.conf
```
But it seems still work on MU... That is what I don't understand.

Reply:
Maybe because the working directory is set to `/root/`? => <jvmFlag>-Dworking.directory=/root/</jvmFlag>

Relative path `conf/cassandra-driver.conf` from working directory `/root/` is the same as absolute path `/root/conf/cassandra-driver.conf` ? 

Would be my guess :)

Reply:
> Relative path conf/cassandra-driver.conf from working directory /root/ is the same as absolute path /root/conf/cassandra-driver.conf

Theoretically it should work but it was not on my local. And I see a similar fix for that too (fixed by you :p):
`JAMES-3668 Add absolute paths for extra properties file in docker image`